### PR TITLE
add package info

### PR DESCRIPTION
**Pull request by leonardoalt**
wanted for release

### CODE REVIEW

**Comment by leonardoalt** [comment]
the descriptions are still vague and simple but I think that's fine for now

File: powdr/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 [package]
 name = "powdr"
-version = "0.1.0"
-edition = "2021"
+description = "powdr crate to be used externally"
```
Comment (lvella) [inline powdr/Cargo.toml:3]:
```suggestion
description = "main powdr crate, reexporting the key APIs"
```

Comment (lvella) [reply powdr/Cargo.toml:3]: I was confused by the description when I read it the first time. Maybe this make it better?

**Review by lvella — DISMISSED** [review]

File: asm-to-pil/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 [package]
 name = "powdr-asm-to-pil"
-version = "0.1.0"
-edition = "2021"
+description = "powdr asm to PIL compiler"
```
Comment (chriseth) [inline asm-to-pil/Cargo.toml:3]: This crate transforms vm machines to constrained machines, i.e. translates the asm instructions to constraints. The result is PIL in the same way as "fully analyzed asm" is pil.

File: number/Cargo.toml [hunk 3]
```diff
@@ -1,7 +1,11 @@
 [package]
 name = "powdr-number"
-version = "0.1.0"
-edition = "2021"
+description = "powdr field element definitions"
```
Comment (chriseth) [inline number/Cargo.toml:3]:
```suggestion
description = "powdr finite field definitions"
```

File: asm-utils/Cargo.toml [hunk 4]
```diff
@@ -1,7 +1,11 @@
 [package]
 name = "powdr-asm-utils"
-version = "0.1.0"
-edition = "2021"
+description = "powdr asm utilities"
```
Comment (chriseth) [inline asm-utils/Cargo.toml:3]: These are utilities for riscv- (and in the future valida / llvm) assembly. Not sure if we should clarify

File: importer/Cargo.toml [hunk 5]
```diff
@@ -1,7 +1,11 @@
 [package]
 name = "powdr-importer"
-version = "0.1.0"
-edition = "2021"
+description = "powdr library import system"
```
Comment (chriseth) [inline importer/Cargo.toml:3]:
```suggestion
description = "powdr module import system"
```

File: asm-utils/Cargo.toml (hunk 4 above)
Comment (leonardoalt) [reply asm-utils/Cargo.toml:3]: I'll add some more words

**Review by chriseth — APPROVED** [review]

**Pull request merged**